### PR TITLE
Convert Homepage into React Components

### PR DESCRIPTION
**Pull request by VLeads**
Made component of each part of Homepage.

Files to review: 
- src/components/card/category-card/CategoryCard.jsx
- src/components/card/product-card/ProductCard.jsx
- src/data/category.data.js
- src/data/homepageProduct.data.js
- src/pages/homepage/CategoriesSection.jsx
- src/pages/homepage/ChooseUsSection.jsx
- src/pages/homepage/PopularPurchas
- src/pages/homepage/homepage.jsx

you can ignore CSS files

Final lookup:
rapid-store-homepage
eSection.jsx

### CODE REVIEW

File: src/components/card/product-card/ProductCard.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export const ProductCard = ({ details }) => {
+  const {
+    img: { altText, url },
+    heading,
+    price: { ourPrice, originalPrice },
+    badge: { type, tagline },
+    _id,
+  } = details;
+
+  console.log("test", ourPrice);
```
Comment (foolhardy21) [inline src/components/card/product-card/ProductCard.jsx]: you may remove this if not needed ahead.

**Review by foolhardy21 — APPROVED** [review]
good use of components breakdown.

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -1,57 +1,55 @@
...
+          <h4>Help</h4>
+          <ul className="footer__list">
+            <li className="footer__list-item">
+              <a href="#">FAQ</a>
+            </li>
+            <li className="footer__list-item">
+              <a href="#">Privacy Policy</a>
+            </li>
+          </ul>
+        </div>
```
Comment (Maianki) [inline src/components/footer/Footer.jsx:32]: Suggestion: Here the structure of the code is similar so what you can do is put the link and corresponding text in an array of objects and map on that array.

**Review by Maianki — APPROVED** [review]
Nice work Vishal. Has left one suggestion.
Things I liked about your code:
- Good naming convention for components 
- Code is readable and clean

File: src/components/footer/Footer.jsx (hunk 2 above)
Comment (VLeads) [reply src/components/footer/Footer.jsx:32]: yes, I'll make these changes in future :)

**Pull request merged**